src/app_service/service/transaction/service.nim

```diff
@@ -261,7 +261,7 @@ QtObject:
   ) =
     try:
       var paths: seq[TransactionBridgeDto] = @[]
-      let amountToSend = conversion.eth2Wei(parseFloat(value), 18)
+      let amountToSend = value.parse(Uint256)
```

Comment:
does that require change in status go?

Reply:
For this change - no. In general taking raw amount from go is needed to e.g. reliably compare values